tests/util/mod.rs

```diff
@@ -181,12 +181,10 @@ pub fn order(point: EdwardsPoint) -> &'static str {
         } else {
             "8"
         }
+    } else if point.is_torsion_free() {
+        "p"
     } else {
-        if point.is_torsion_free() {
-            "p"
-        } else {
-            "8p"
-        }
+        "8p"
```

Comment:
Pre-existing potential bug: $2p$ and $4p$ are also possible, aren't they?

Reply:
Good catch, I created https://github.com/ZcashFoundation/ed25519-zebra/issues/83 to fix it later